node_modules/oae-admin/tenants/tenants.html

```diff
@@ -121,8 +42,99 @@
             </div>
         </div>
     {/if}
+    {var isOnGlobalAdminServer = context.isGlobalAdminServer || context.isTenantOnGlobalAdminServer}
+    {if context.isGlobalAdminServer}
+        <div id="tenants-search">
+            <form action="." role="search" aria-label="__MSG__SEARCH__">
+                <label for="tenants-search-query" class="sr-only">__MSG__NARROW_BY_KEYWORD__</label>
+                <input type="text" id="tenants-search-query" class="form-control" title="__MSG__NARROW_BY_KEYWORD__" autofocus placeholder="__MSG__NARROW_BY_KEYWORD__" />
+            </form>
+        </div>
+    {/if}
+    <div id="tenants-table">
+        <ul id="tenants-table-header" class="oae-list">
+            <li class="row">
+                <div class="tenants-col tenants-col-edit"></div>
+                <div class="col-xs-1">__MSG__ALIAS__</div>
+                <div class="col-xs-2">__MSG__NAME__</div>
+                <div class="col-xs-4">__MSG__HOST__</div>
+                <div class="col-xs-1">__MSG__STATUS__</div>
+            </li>
+        </ul>
+        <ul id="tenants-table-rows" class="oae-list"></ul>
+    </div>
+--></div>
+
+<div id="tenants-row-template"><!--
+    {var isOnGlobalAdminServer = context.isGlobalAdminServer || context.isTenantOnGlobalAdminServer}
+    {for tenant in results}
+        {var displayName = oae.api.util.security().encodeForHTMLAttribute(tenant.displayName)}
+        <li id="tenants-table-row-${tenant.alias|encodeForHTMLAttribute}" data-alias="${tenant.alias|encodeForHTMLAttribute}" class="row">
```

Comment:
If we already have data-alias, is it necessary to incorporate the alias into the id as well?